src/components/InviteParticipants/InviteParticipants.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import InviteForm from './InviteForm'
+import Modal from '../Modal/Modal'
+import Button from '@material-ui/core/Button'
+import { selectToken } from '../../utils/selectors'
+import { useSelector } from 'react-redux'
+import { Api } from '../../services/api'
+
+const InviteParticipants = () => {
+    const token = useSelector(selectToken)
+    const [isModalOpened, setIsModalOpened] = useState(false)
+
+    const onSubmit = (values, { setSubmitting }) => {
+        const { participantAddresses } = values
+        const addresses = participantAddresses.split(' ').map((email) => ({
+            email,
+            lang: 'fr',
+        }))
+        console.log('On submit called: ', addresses)
+
+        try {
+            const api = new Api(token)
```

Comment:
I believe this should be in an redux action, where we request to store to get the token.

Reply:
and I can set the `api` instance within the `store` ?
It would be awesome indeed to instance this class after obtaining a token and make it available to the rest of the application

Reply:
- the user token is already stored in `redux` (ok to put it in a parent `user` or `token` object
- I think async actions allow to use `dispatch` in a single async function, but I'm not using dispatch here

Reply:
I understand dispatch is not used, but the invitation of users should be an action dispatched from the component. Even if it does not modify the local or global state.

Reply:
why ? In that case, I have the impression that every async function used in a component might be turned into an `async action`

Reply:
now that I think about this, if I use a single loader for the app (that overlays the whole UI), I might dispatch the action to display the loader, and thus from within the `inviteParticipants` method, and thus I see a perfect case to move that into an async action ;)

Reply:
Yes exactly
- have the same architecture for all component: they want to trigger an action on something, go through redux
- provide better tracking of what is going through within the app with Redux tooling
- separate Component from business side: Component are responsible for the UI and routing, action manage async behavior, network request and business rules. The component just want to invite users, it doesn't care how to find the token, the action will do it for him.

Reply:
#336